clients/mssql/store.go

```diff
@@ -75,11 +74,10 @@ func (s *Store) Dedupe(_ sql.TableIdentifier, _ []string, _ kafkalib.TopicConfig
 }
 
 func (s *Store) GetTableConfig(tableData *optimization.TableData) (*types.DwhTableConfig, error) {
-	// TODO: Figure out how to leave a comment.
 	const (
 		describeNameCol        = "column_name"
 		describeTypeCol        = "data_type"
-		describeDescriptionCol = "description"
+		describeDescriptionCol = "column_default"
```

Comment:
Is this using `ColumnDescLabel` field as the default value for MSSQL and the column description for other databases types? Can we use different fields so we don't confuse the two?

Reply:
Yes it is. I'll rename

Reply:
Err, slight correction after reading this for the second time.

We're querying this field to see what the default value is.